src/WarcraftLegacies.Source/Quests/Lordaeron/QuestCapitalCity.cs

```diff
@@ -56,11 +58,12 @@ protected override void OnFail(Faction completingFaction)
     /// <inheritdoc/>
     protected override void OnComplete(Faction completingFaction)
     {
-      completingFaction.Player?.RescueGroup(_rescueUnits);
+      if (completingFaction.Player != null)
+        completingFaction.Player.RescueGroup(_rescueUnits);
       SetUnitInvulnerable(_unitToMakeInvulnerable, true);
       if (GetLocalPlayer() == completingFaction.Player)
         PlayThematicMusic("war3mapImported\\CapitalCity.mp3");
-      LegendLordaeron.Uther?.AddUnitDependency(LegendLordaeron.CapitalPalace.Unit);
```

Comment:
I think we probably want to keep the null check here, in case this quest is accidentally set up before Lordaeron is.
That said we should find a way to make it impossible for Uther to be null here. Maybe dependency injection?